ts/components/services/ServiceMetadata/InformationRow.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { View, StyleSheet } from "react-native";
+import { View as NBView } from "native-base";
+
+import { TranslationKeys } from "../../../../locales/locales";
+import { capitalize } from "../../../utils/strings";
+import I18n from "../../../i18n";
+import { H4 } from "../../core/typography/H4";
+import TouchableDefaultOpacity from "../../TouchableDefaultOpacity";
+import ItemSeparatorComponent from "../../ItemSeparatorComponent";
+
+const styles = StyleSheet.create({
+  row: {
+    flexDirection: "column"
+  },
+  touchable: {
+    flexDirection: "row",
+    marginVertical: 24
+  },
+  label: {
+    flex: 1
+  },
+  alignToRight: {
+    textAlign: "right"
+  }
+});
+
+type Props = {
+  value: string;
+  label: TranslationKeys;
+  onPress: () => void;
+  hint: TranslationKeys;
+};
+
+const InformationRow = ({ value, label, onPress, hint }: Props) => (
+  <View style={styles.row}>
```

Comment:
```suggestion
  <View>
```
Maybe it is not needed